### DISCUSSION TITLE
sync issues in lineage os

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#23048] [Q&A] [answered]
https://cirrus-ci.com/task/4692181953282048?logs=build#L70

trees are mentioned in local manifest but still its showing dt doesnt exist

**Comment by Apon77** [comment, marked as answer]
no idea why this happened! now there is other sources, and no xiaomi folder at all in the server

**Reply by SwapnilVicky** [reply]
okay i'll try once again
